ods_ci/tests/Resources/Files/pipeline-samples/v2/cache-disabled/gpu/pytorch/pytorch_amd_gpu_availability.py

```diff
@@ -0,0 +1,54 @@
+from kfp import compiler, dsl, kubernetes
+from kfp.dsl import PipelineTask
+
+#  Runtime: Pytorch with ROCm and Python 3.9 (UBI 9)
+common_base_image = (
+    "quay.io/modh/runtime-images@sha256:76d2a5d9ca0bb35c4d9962a7ee6798672afd3a5c7d226d87ef65f952e7cb93cf"
+)
+
+
+def add_gpu_toleration(task: PipelineTask, accelerator_type: str, accelerator_limit: int):
+    print(f"Adding GPU tolerations: {accelerator_type}({accelerator_limit})")
+    task.set_accelerator_type(accelerator=accelerator_type)
+    task.set_accelerator_limit(accelerator_limit)
+    kubernetes.add_toleration(task, key=accelerator_type, operator="Exists", effect="NoSchedule")
+
+
+@dsl.component(
+    base_image=common_base_image
+)
+def verify_gpu_availability(gpu_toleration: bool):
+    import torch
+
+    cuda_available = torch.cuda.is_available()
+    device_count = torch.cuda.device_count()
+    print("------------------------------")
+    print("GPU availability")
+    print("------------------------------")
+    print(f"cuda available: {cuda_available}")
```

Comment:
I think it make sense to replace cuda with rocm

Reply:
PyTorch does not have a separate device string for ROCm, which is why you use 'cuda' to refer to an AMD GPU under ROCm.

Reply:
I know that and that is why i am suggesting to change the print statement with rocm instead of cuda so that people can differentiate easily.

Reply:
imho, if we are getting "cuda" we should print "cuda". in this case, we don't "hide" any detail.
also, having cuda string for rocm is super confusing.